st2common/st2common/script_setup.py

```diff
@@ -60,6 +62,7 @@ def setup(config, setup_db=True, register_mq_exchanges=True):
     2. Establishes DB connection
     3. Suppress DEBUG log level if --verbose flag is not used
     4. Registers RabbitMQ exchanges
+    5. Registers internal trigger types (optional, disabled by default)
```

Comment:
Maybe it worth to register trigger types only when `--register-trigger` is requested from the bootstrap script?

Move this logic here: https://github.com/StackStorm/st2/blob/master/st2common/st2common/content/bootstrap.py#L128 from the `setup`.